Makefile

```diff
@@ -175,6 +180,11 @@ docker-build-bundle: bundle
 docker-push-bundle:
 	docker push ${BUNDLE_IMG}
 
+.PHONY: docker-build-tools
+docker-generate-protobuf: ./build/Containerfile.tools
+	buildah bud -f $^ -t ${TOOLS_IMG} .
+	podman run --rm -ti --volume=${PWD}:/go/src/github.com/csi-addons/kubernetes-csi-addons:Z ${TOOLS_IMG} make generate-protobuf
```

Comment:
Not all systems will have buildah and podman and we use docker in most places in this Makefile to run commands.

Reply:
i that would be a good option as some developers might also not use Fedora.

Reply:
Done, please have a look again.